hermes-console-vue/src/views/topic/subscriptions-list/SubscriptionsList.vue

```diff
@@ -25,27 +25,30 @@
 
   const subscriptionItems = computed(() =>
     props.subscriptions
+      ?.filter((subscription) =>
```

Comment:
Previous approach in which we mapped either to object (name/color/statusText/href) or to `null` and then filtered with `.filter(Boolean)` was problematic for TS type system. Though the operation filtered out null objects, IntelliJ still complained in Vue template below, that `subscriptionItems` elements can be possibly `null`.

We could introduce an interface:
```typescript
interface SubscriptionListItem {
  name: string;
  color: string;
  statusText: State;
  href: string;
} 
```
and then change `.filter(Boolean)` filter to the so-called TypeScript type predicate:
```typescript
.filter((subscription): subscription is SubscriptionListItem => subscription !== null)
```
then TS type system would know, that null values possibly introduced in `map` above were filtered out.

But IMHO it is cleaner to simply perform subscription filtering before `map`, as I did here